src/main/java/org/dependencytrack/model/Tag.java

```diff
@@ -64,6 +66,7 @@ public class Tag implements Serializable {
     private String name;
 
     @Persistent
+    @JsonBackReference
```

Comment:
`@JsonIgnore` may be a better choice here. `@JsonBackReference` has some special behavior for deserialization, for which another field annotated with `@JacksonManagedReference` is required. It doesn't appear like we ever need to deserialize the `projects` field of `Tag`.